docs/migration-guide/to-16.md

```diff
@@ -12,3 +12,20 @@ Node.js 14 has reached end-of-life. We've removed support for it so that we coul
 
 - 16.13.0
 - 18.0.0
+
+## Changed CLI behavior
+
+Starting from 16.0.0, the CLI prints problems to stderr instead of stdout.
+
+If you use the CLI to fix a source string by using the [`--fix`](../user-guide/cli.md#--fix) and [`--stdin`](../user-guide/cli.md#--stdin) options, the CLI will print the fixed code to stdout and any problems to stderr.
+
+## Changed Node.js API behavior
+
+The Promise returned by `stylelint.lint()` has the following changes:
+
+- The [output](../user-guide/node-api.md#output) property of the resolved object now only contains formatted problems.
+- A new [code](../user-guide/node-api.md#code-1) property which contains the autofixed code has been added to the resolved object.
```

Comment:
```suggestion
## Changed Node.js API returned resolved object

We changed the resolved object of the Promise returned by `stylelint.lint()` so that:

- the [output](../user-guide/node-api.md#output) property now only contains formatted problems
- a new [code](../user-guide/node-api.md#code-1) property contains the autofixed code
```